### PR TITLE
feat: Add diag/sys command

### PR DESCRIPTION
**Pull request by Dignifiedquire**

### CODE REVIEW

**Comment by diasdavid** [comment]
thank you :) - released on https://github.com/ipfs/js-ipfs-api/releases/tag/v2.9.1

File: src/index.js [hunk 1]
```diff
@@ -123,7 +123,8 @@ function IpfsAPI (host_or_multiaddr, port) {
   }
 
   self.diag = {
-    net: command('diag/net')
+    net: command('diag/net'),
```
Comment (jbenet) [inline src/index.js]: maybe remove this command from the api? we'll be deprecating it

Comment (diasdavid) [inline src/index.js]: on 0.3.10? We can deprecate it when that IPFS version is released.

Comment (Dignifiedquire) [inline src/index.js]: Why is it deprecated? :( It has some nice details, is there an alternative to get this data?

**Pull request merged**